src/index.js

```diff
@@ -74,6 +75,11 @@ function validateOptions(options) {
       throwInvalid(`Invalid option — ${option} must be an integer.`);
     }
   });
+
+  // Invalid cache dir (not an absolute path)
+  if ('cacheDir' in options && !path.isAbsolute(options.cacheDir)) {
+    throwInvalid(`Provided cacheDir is not absolute: '${cacheDir}'`);
```

Comment:
This line is causing a build failure due to a linting error: `error  'cacheDir' is not defined`

Reply:
Good catch!